docs/general/serializers.md

```diff
@@ -217,7 +217,15 @@ The object being serialized.
 
 #### #root
 
-PR please :)
+Allows you to change the root object name of the payload.
+
+e.g.
+
+```ruby
+def root
+  "posts"
+end
```

Comment:
@jimbeaudoin Thanks for this!  mind taking a look at https://github.com/rails-api/active_model_serializers/pull/1782 and see what you think? cc @beauby